proposals/12147-mlmd-removal/protos/runs.json

```diff
@@ -0,0 +1,144 @@
+{
+  "experiment_id": "f7344db6-de5d-4e68-816b-98b4f0d1ca7f",
```

Comment:
I think the runs namespace is missing in the example but it might be good to include to show the artifacts are in the same namespace.